opensearch/services/datasets/datasets.php

```diff
@@ -162,6 +178,19 @@ public function getSearchResultOutput(array $result) {
 			$html .= "</span>";
 		}
 
+		//data types
```

Comment:
I would say leave it in, but we should evaluate how it will work because in looking at it now I think it needs to be tweaked.

1) we need to call getDataTypesByDataset( ) to run the query to return the set of data types for the dataset
```
$datatypes = $this->getDataTypesByDataset($result['dataset']);
```
2) loop through $datatypes and add them to the html as a link (see datasets.php:195 or 214 as examples)